backend/geonature/core/gn_meta/routes.py

```diff
@@ -103,6 +103,8 @@ def get_datasets():
     :returns:  `list<TDatasets>`
     """
     params = MultiDict(request.args)
+    if request.is_json:
+        params.update(MultiDict(request.json))
```

Comment:
```suggestion
        params.update(request.json)
```
`update` accepte un mapping donc un dict